### PR TITLE
Update FastLSTM.lua

### PR DESCRIPTION
**Pull request by supakjk**
"p" and "mono" should be processed in the parent (LSTM) initialization.

### CODE REVIEW

**Comment by supakjk** [comment]
After modifying the initialization of FastLSTM, Bayesian dropout seems work but it's much slower than adding an external dropout layer before or after FastLSTM. Do you have any idea why it would be so slow?

File: FastLSTM.lua [hunk 1]
```diff
@@ -160,7 +155,6 @@ function FastLSTM:nngraphModel()
       h2h = bn_wh(self.o2g(prev_h):annotate{name='h2h'}):annotate {name = 'bn_wh'}
       
       -- add bias after BN as per paper
-      self.o2g:noBias()
```
Comment (jnhwkim) [inline FastLSTM.lua:163]: Why did you remove this line of code?

**Comment by jnhwkim** [comment]
Did you check if cuda computing is enabled? I believe, usually, dropouts don't slow down a single iteration.

**Comment by jnhwkim** [comment]
@supakjk this repository is experimental one. Please send a pull request to [Element-Research/rnn](https://github.com/Element-Research/rnn).
